kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -671,6 +671,13 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(true)
 
+  val FRONTEND_THRIFT_STOP_TIMEOUT: ConfigEntry[Long] =
+    buildConf("kyuubi.frontend.thrift.stop.timeout")
+      .doc("Timeout for stop the thrift frontend service.")
+      .version("1.7.0")
+      .timeConf
+      .createWithDefault(Duration.ofSeconds(60).toMillis)
```

Comment:
Does it mean we can not gracefully shutdown queries that requires more than this time to complete?